internal/runnerinstall/k8s.go

```diff
@@ -243,6 +390,7 @@ type K8sConfig struct {
 	CpuRequest           string
 	MemRequest           string
 	CreateServiceAccount bool
+	KeepPV              bool
```

Comment:
Is there anything in the persistent volume that the user might want to keep after uninstalling?

Reply:
Probably not, I only thought about adding this in because helm won't delete PVs regardless of uninstall flags and leaves it up to the cluster to reap them after a period of time which an operator can configure. I'll go ahead and remove it since its really just for storing that token which really has no use after the runner is uninstalled.